composer.json

```diff
@@ -21,10 +21,13 @@
     "require-dev": {
         "doctrine/doctrine-bundle": "^1.8 || ^2.0",
         "doctrine/phpcr-odm": "^1.4",
+        "jackalope/jackalope": "^1.0",
+        "jackalope/jackalope-doctrine-dbal": "^1.0",
+        "jackalope/jackalope-jackrabbit": "^1.0",
```

Comment:
i would only install one of dbal or jackrabbit. and you don't need to require jackalope/jackalope separately, it is required by the actual implementations.

as to why this suddenly comes up: maybe old versions of symfony-cmf/testing did require jackalope but the newest one no longer does? you'd need to compare composer.json of symfony-cmf/testing if you want to be sure.

but imho requiring something explicitly here is a good idea anyways. you might also need to do a bit of setup in the ci steps to be able to use the implementation if there are functional tests in this repository.

Reply:
i required the `jackalope-doctrine-dbal` for now. Maybe @dantleech can tell us if something specific is required, from my side ready for review :)